modules/jenkins.test.js

```diff
@@ -0,0 +1,111 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const jenkins = require("./jenkins");
+const fs = require("fs");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Tests
+// -----------------------------------------------------------------------------------------
+
+describe("jenkins", () => {
+    describe("configureCredentials", () => {
```

Comment:
Minor - please wrap each of the describe blocks for the test methods in a region block